packages/file-pipeline/src/types.ts

```diff
@@ -16,6 +25,22 @@ export interface FileCacheInterface extends AbstractFileCache {
   toPaths: () => string[]
 }
 
+export interface RouteCacheInterface extends AbstractFileCache {
+  delete(file: File): void
+  add(file: File, type: RouteType): void
+
+  get(): Record<string, RouteCacheEntry>
+  get(key: string): RouteCacheEntry
```

Comment:
should we list: `get(key?: string)` also?

Reply:
Correct me if I am mistaken. Is `get(key?: string)` equal to `get()` if `key === undefined`?

Reply:
yeap, that sounds correct to me!